### ISSUE TITLE
Transform Content-Type application/vnd.api+json

### ISSUE THREAD

**Issue by janwerkhoven** [vigetlabs/olive_branch#27]
Currently transforming only happens on requests with Content-Type `application/json`.

For those following the [JSON API standard](https://jsonapi.org/) the Content-Type is `application/vnd.api+json`.

Those people can now force transforms with:

```
config.middleware.use OliveBranch::Middleware, content_type_check: -> (content_type) { true }
```

Perhaps Olive Branch should default transforming Content-Type `application/vnd.api+json` as well?

**Comment by efatsi** [comment]
👍 great call. Just released version 2.1.4 handling this.

**Comment by janwerkhoven** [comment]
Thanks!
